docker/test/integration/minifi/core/KubernetesRunner.py

```diff
@@ -0,0 +1,55 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import logging
+import os
+import shutil
+from .KindProxy import KindProxy
+from .MinifiContainer import MinifiContainer
+
+
+class KubernetesRunner(MinifiContainer):
```

Comment:
I think this should be renamed to reflect that this runs a minifi container inside a kubernetes environment, KubernetesRunner seems too generic opposed to the function of the class.

Reply:
I have renamed the class to `MinifiAsPodInKubernetesCluster` in https://github.com/apache/nifi-minifi-cpp/pull/1270/commits/26d204d8759b5f9b5a61e9157d4bffbac217d9d1.